09-Photutils/photutils_overview.ipynb

```diff
@@ -1241,7 +1241,20 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`catalog` is a `SourceCatalog` object.  It behaves like a list of `SourceProperties` objects, one for each source."
+    "`catalog` is a `SourceCatalog` object.  It behaves like a list of `SourceProperties` objects, one for each source.\n",
+    "\n",
+    "The `to_table` method converts the `SourceCatalog` object to a `QTable`. We will do this so we can write the table to disk for future use, but going forward in this example we will use the `SourceCatalog` object for analysis of the detected objects."
```

Comment:
```suggestion
    "The `to_table` method converts a subset of the `SourceCatalog` object properties (e.g., the scalar properties) to a `QTable`. We will do this so we can write the table to disk for future use, but going forward in this example we will use the `SourceCatalog` object for analysis of the detected objects."
```

Reply:
Not all `SourceCatalog` properties are put in the table (e.g., cutout 2D images).